### PR TITLE
Add domains to sites

### PR DESCRIPTION
**Pull request by ITViking**
#### What does this PR do?
This PR add all the known domain for libraries to sites.yaml. 

#### Should this be tested by the reviewer and how?
Just read the PR through and make sure it is sane

#### What are the relevant tickets?
https://reload.atlassian.net/jira/software/c/projects/DDFDRIFT/boards/464?selectedIssue=DDFDRIFT-120 & https://reload.atlassian.net/jira/software/c/projects/DDFDRIFT/boards/464?selectedIssue=DDFDRIFT-105

### CODE REVIEW

File: infrastructure/environments/dplplat01/sites.yaml [hunk 1]
```diff
@@ -306,21 +480,39 @@ sites:
...
   koge:
     name: "KøgeBibliotekerne"
     description: "The library site for Køge"
     deploy_key: "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIMRB54Z7o6Vt3qOucjvE7No6LN2NdBNtfRi2f4y+UQdA"
     <<: *default-release-image-source
+    primary-domain: "koegebib.dk"
+    secondary-domains:
+      - "www.koegebib.dk"
+      - "køgebib.dk"
+      - "www.køgebib.dk"
```
Comment (hypesystem) [inline infrastructure/environments/dplplat01/sites.yaml:499]: Vi har ikke testet om det her virker eller om det skal punycodes

**Review by hypesystem — COMMENTED** [review]
Har nogle kommentarer jeg ikke ved om skal løses eller ej?

File: infrastructure/environments/dplplat01/sites.yaml (hunk 1 above)
Comment (ITViking) [reply infrastructure/environments/dplplat01/sites.yaml:505]: jeg prøvet teste det her nu

Comment (hypesystem) [reply infrastructure/environments/dplplat01/sites.yaml:505]: Skal ændres til punycoded url for at virke

**Review by hypesystem — CHANGES_REQUESTED** [review]
Én sidste ændring før vi er good-to-go

**Comment by ITViking** [comment]
Good question - i have no answer for that right now. But we should probably look into it

**Pull request closed without merging**